.circleci/config.yml

```diff
@@ -192,7 +192,9 @@ jobs:
           steps:
             - run:
                 name: Read version number
-                command: echo 'export VERSION_NAME=$(cat version.json | jq .versionName)' >> $BASH_ENV
+                command: |
+                  echo "export VERSION_NAME=$(cat version.json | jq .versionName)" >> $BASH_ENV
+                  source $BASH_ENV
```

Comment:
This line can be removed. The BASH_ENV is sourced in each step

Reply:
Will be removed in https://github.com/Integreat/integreat-webapp/pull/352